ajax/libs/iris-ws/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "iris-ws",
+  "title": "iris-ws",
+  "filename": "client.js",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/kataras/iris.git",
+    "fileMap": [
+      {
+        "basePath": "websocket",
+        "files": [
+          "client.js",
+          "client.min.js"
+        ]
+      }
+    ]
+  },
+  "description": "Websocket client-side library for the golang web-framework called iris.",
```

Comment:
May I know the reference of this description?

Reply:
I've wrote this line myself. If you feel like, you can take a look at the original web-socket library (which has been used as it is, in the **IRIS** web-framework) here: https://github.com/kataras/go-websocket